packages/forms/src/components/Repeater/Repeater.tsx

```diff
@@ -145,95 +178,143 @@ const Fieldset = ({
+              data-h2-width="base(100%)"
+              data-h2-margin-bottom="base(x.5)"
             >
-              <ActionButton
-                disabled={disabled || index <= 0}
-                onClick={decrement}
-                decrement
-                animate={!shouldReduceMotion}
-                aria-label={intl.formatMessage(formMessages.repeaterMove, {
-                  from: position,
-                  to: position - 1,
-                })}
+              <div
+                data-h2-display="base(flex)"
+                data-h2-align-items="base(center)"
+                data-h2-margin-left="base(-x.5)"
               >
-                <ChevronUpIcon data-h2-width="base(x1)" />
-              </ActionButton>
-              {!hideIndex && (
-                <span
-                  aria-hidden="true"
-                  data-h2-text-align="base(center)"
-                  data-h2-font-weight="base(700)"
-                  data-h2-margin="base(x.25, 0)"
-                >
-                  {index + 1}
-                </span>
-              )}
-              <ActionButton
-                disabled={disabled || index === total - 1}
-                onClick={increment}
-                animate={!shouldReduceMotion}
-                aria-label={intl.formatMessage(formMessages.repeaterMove, {
-                  from: position,
-                  to: position + 1,
-                })}
+                {!isLocked ? (
+                  <>
+                    <ActionButton
+                      disabled={isFirstItem || previousItemLocked}
+                      onClick={decrement}
+                      decrement
+                      animate={!shouldReduceMotion}
+                      aria-label={intl.formatMessage(
+                        formMessages.repeaterMove,
+                        {
+                          from: position,
+                          to: position - 1,
+                        },
+                      )}
+                      data-h2-font-weight="base(700)"
+                      data-h2-margin="base(0, x.25, 0, 0)"
+                      {...(isFirstItem
+                        ? { "data-h2-padding-left": "base(x.25)" }
+                        : {})}
+                    >
+                      {!isFirstItem && !previousItemLocked ? (
+                        <ArrowUpIcon data-h2-width="base(x.75)" />
+                      ) : (
+                        <div data-h2-width="base(x.75)">
+                          <p data-h2-padding-bottom="base(x.25)">.</p>
+                        </div>
+                      )}
+                    </ActionButton>
+                    {!hideIndex && (
+                      <span
+                        aria-hidden="true"
+                        data-h2-text-align="base(center)"
+                        data-h2-font-weight="base(700)"
+                        data-h2-margin="base(x.25, 0)"
+                      >
```

Comment:
I'd add some padding here or use the gap property on the parent element to help spread some items out a big. They seem a little bit close right now.

![Screenshot 2023-10-13 120150](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/7955fbc1-a646-464b-885b-be0012a7736a)

Reply:
Sounds good 👍 Fixed by [e51e6ab](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8181/commits/e51e6ab11f1085af2d7a5fa68c9f549426c83db3).